packages/fe/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@ui5-language-assistant/fe",
+  "version": "3.3.1",
+  "description": "UI5 LSP server extension for annotation relevant building blocks",
+  "keywords": [],
+  "files": [
+    ".reuse",
+    "LICENSES",
+    "lib/src",
+    "api.d.ts",
+    "src"
+  ],
+  "main": "lib/src/api.js",
+  "repository": "https://github.com/sap/ui5-language-assistant/",
+  "license": "Apache-2.0",
+  "typings": "./api.d.ts",
+  "dependencies": {
+    "lodash": "4.17.21"
+  },
+  "devDependencies": {
+    "@sap-ux/vocabularies-types": "0.6.8",
+    "@types/lodash": "4.14.166",
+    "@ui5-language-assistant/context": "3.3.1",
+    "@ui5-language-assistant/logic-utils": "3.3.1",
+    "@ui5-language-assistant/semantic-model-types": "3.3.1",
+    "@ui5-language-assistant/test-framework": "3.3.1",
+    "@ui5-language-assistant/test-utils": "3.3.1",
+    "@ui5-language-assistant/xml-views-completion": "3.3.1",
+    "@ui5-language-assistant/xml-views-validation": "3.3.1",
+    "@xml-tools/ast": "5.0.0",
+    "@xml-tools/parser": "1.0.7",
+    "vscode-languageserver-types": "3.17.2",
+    "vscode-languageserver-protocol": "3.17.2"
+  },
+  "scripts": {
+    "ci": "npm-run-all clean compile lint coverage",
+    "clean": "rimraf ./lib ./coverage ./nyc_output",
+    "compile": "yarn run clean && tsc -p .",
+    "compile:watch": "tsc -p . --watch",
+    "lint": "eslint . --ext .ts --max-warnings=0 --ignore-path=../../.gitignore",
+    "test": "mocha",
+    "coverage": "nyc mocha",
+    "format:fix": "prettier --write \"**/*.@(js|ts|json|md)\" --ignore-path=.gitignore"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
will you make these modules `private` in the future?

Reply:
Probably not. I don't see reasons why we should do it